README.md

```diff
@@ -144,24 +153,31 @@ The returned object from an execute call possesses the following methods:
 }
 ```
 
-#### Usage in a Web Browser
+### Use in a web browser
 
-The signalflow client can be built for usage in a browser. This is accomplished via the following commands.
+The signalflow client can be built for use in a browser. 
 
-```
-$ npm install
-$ npm run build:browser
-The output can be found at ./build/signalflow.js
-```
+1. Run the following commands:
 
-It can then be loaded as usual via a script tag
+   ```
+   npm install
+   npm run build:browser
+   ```
 
-```
-<script src="build/signalflow.js" type="text/javascript"></script>
-```
+   Expected output:
+
+   ```
+   The output can be found at ./build/signalflow.js
+   ```
+
+2. Load the client using a `script` tag:
+
+   ```
+   <script src="build/signalflow.js" type="text/javascript"></script>
+   ```
 
-Once loaded, signalfx global will be created(window.signalfx). Note that only the SignalFlow package is included in this built file.
+   Once loaded, signalfx global is created(window.signalfx). Note that the built file only includes the SignalFlow package.
```

Comment:
What is meant by "created(window.signalfx)"? Once I understand this, I might have a suggestion for how to convey it differently. Or maybe this is the best way to say it and I just don't know the convention! : )

Reply:
It should've been `created (window.signalfx)`, meaning in JavaScript a globally available `signalfx` object in the browser is created / made available.